tutorials/cloud/microstack-get-started/microstack-get-started.md

```diff
@@ -5,157 +5,81 @@ categories: cloud
+[OpenStack][openstack-upstream] is a collection of open source projects designed to work together to form the basis of a cloud. OpenStack can be used for both private and public clouds.
 
 ### What is MicroStack?
 
-[MicroStack][microstack] is an upstream single-node OpenStack deployment which can run directly on your workstation. Although made for developers, it is also suitable for edge, IoT and appliances. MicroStack is an OpenStack in a [snap][snapcraft] which means that all OpenStack services and supporting libraries are packaged together in a single package which can be easily installed, upgraded or removed. MicroStack includes all key OpenStack components: Keystone, Nova, Neutron, Glance, Cinder, and is evolving extremely fast. You can use it for development, prototyping and testing. MicroStack is perfectly suitable for the network edge, IoT and appliances. Let's get started - grab MicroStack from the Snap Store and get your OpenStack running right away!
+[MicroStack][microstack] provides a single or multi-node OpenStack deployment which can run directly on your workstation. Although made for developers to prototype and test, it is also suitable for edge, IoT, and appliances.  MicroStack is an OpenStack in a [snap][snapcraft] which means that all OpenStack services and supporting libraries are packaged together in a single package which can be easily installed, upgraded or removed. MicroStack includes all key OpenStack components: Keystone, Nova, Neutron, Glance, and Cinder.
 
 ### In this tutorial you will learn how to:
 
-- Get your OpenStack cluster up and running with MicroStack
+- Get a single-node OpenStack cloud up and running with MicroStack
 - Interact with OpenStack via the web GUI and CLI
 - Launch your first VM on OpenStack and access it
 
 ### You will need:
 
-* A machine with Linux, a multi-core processor and at least 16 GB of RAM
-
-
+* A machine with Linux, a multi-core processor and at least 8 GB of RAM
 
 ## Install MicroStack
 
-Duration: 5:00
+Duration: 2:00
 
-If you are using Ubuntu Xenial or later, which we highly recommend, you can install MicroStack right away by running the following command from the terminal:
+If you are using Ubuntu Xenial or later, which we highly recommend, you can install MicroStack right away. Here we are installing from the beta channel:
```

Comment:
Right now, MicroStack is only tested on Bionic. People will run into trouble in Xenial, and may run into trouble in some of the newer interim releases.